src/dhtproto/client/mixins/NeoSupport.d

```diff
@@ -1291,7 +1291,42 @@ template NeoSupport ( )
 
     /***************************************************************************
 
-        Helper function to initialise neo components.
+        Helper function to initialise neo components. Automatically calls
+        addNodes() with the node definition files specified in the Config
+        instance.
+
+        Params:
+            config = swarm.client.model.IClient.Config instance. (The Config
+                class is designed to be read from an application's config.ini
+                file via ocean.util.config.ClassFiller.)
+            conn_notifier = delegate which is called when a connection attempt
+                succeeds or fails (including when a connection is
+                re-established) and when the hash-range for a node is first
+                queried. Of type:
+                void delegate ( Neo.DhtConnNotification )
+
+    ***************************************************************************/
+
+    private void neoInit ( Neo.Config config,
+        Neo.DhtConnectionNotifier user_conn_notifier )
+    {
+        this.user_conn_notifier = user_conn_notifier;
```

Comment:
The old overload of `neoInit` has a contract asserting that the `user_conn_notifier` must be not-null. Is this not applicable here?

Reply:
Good catch. (I think I originally intended the new one to call the old one, but it didn't work out that way.)